churaverse-plugins-client/src/gamePlugin/gameUiManager.ts

```diff
@@ -0,0 +1,56 @@
+import { GameIds } from './interface/gameIds'
+import { IGameUiComponent } from './interface/IGameUiComponent'
+import { GameUiRegister } from './gameUiRegister'
+import { IGameUiManager } from './interface/IGameUiManager'
+
+export class GameUiManager implements IGameUiManager {
+  private readonly gameUiRegister: GameUiRegister
+
+  public constructor(gameUiRegister: GameUiRegister) {
+    this.gameUiRegister = gameUiRegister
+  }
+
+  /**
+   * 特定のGameIdのUIを初期化する
+   */
+  public initializeAllUis(gameId: GameIds): void {
+    const components = this.getAllUiComponents(gameId)
+    components.forEach((component) => {
+      component.initialize()
+      if (!component.visible) {
+        component.element.style.display = 'none'
+      }
+    })
+  }
+
+  /**
+   * 指定されたゲームidとUI名に対応するUIを取得する
+   * @param gameId ゲームid
+   * @param uiName UI名
```

Comment:
このコメントだと情報量が増えてないので省略してしまってもいい気がします
```suggestion
   * 引数で指定したGameIdとUiNameに対応するUIを取得する
```